content/client-server-api/modules/instant_messaging.md

```diff
@@ -327,11 +327,11 @@ If a reason were to be supplied, it would look like:
 }
 ```
 
-When sending a spoiler, clients SHOULD provide the plain text fallback in the `body`
-as shown above (including the reason). The fallback SHOULD omit the spoiler text verbatim
-since `body` might show up in text-only clients or in notifications. To prevent spoilers
-showing up in such situations, clients are strongly encouraged to first upload the plaintext
-to the media repository then reference the MXC URI in a markdown-style link, as shown above.
+When sending a spoiler, clients SHOULD provide the fallback in the `body` as shown above
+(including the reason). The fallback SHOULD omit the spoiler text verbatim since `body`
+might show up in text-only clients or in notifications. To prevent spoilers showing up in
+such situations, clients are strongly encouraged to first upload the text containing spoilers
+to the media repository, then reference the MXC URI in a markdown-style link, as shown above.
```

Comment:
Apologies for the poor diff due to formatting. The real diff is:

> When sending a spoiler, clients SHOULD provide **the fallback** in the `body` as shown above
> (including the reason). The fallback SHOULD omit the spoiler text verbatim since `body`
> might show up in text-only clients or in notifications. To prevent spoilers showing up in
> such situations, clients are strongly encouraged to first upload **the text containing spoilers**
> to the media repository, then reference the MXC URI in a markdown-style link, as shown above.

Reply:
I don't know what "omit the spoiler text verbatim" means, but that's not new.

This looks like a net clarification to me.

Reply:
Perhaps we can drop the "verbatim". In any case, it just means: don't include the text including spoilers in `body`.

Reply:
I've tried to clean this sentence up a bit in https://github.com/matrix-org/matrix-spec/pull/1306/commits/c24fb1d0e0bae15d40a6411230351809425eb4f8.